script.js

```diff
@@ -0,0 +1,65 @@
+const names = document.getElementById('title');
+const pass = document.getElementById('author');
+const button = document.getElementById('button');
+const list = document.getElementById('list');
+
+list.style.width = '20%';
+
+function Books(title, author) {
+  this.title = title;
+  this.author = author;
+}
```

Comment:
[OPTIONAL] Please consider giving `id` to your books when you initialize them.

<img width="539" alt="Screen Shot 2022-06-27 at 18 04 51" src="https://user-images.githubusercontent.com/82426682/176051243-35da85b8-0188-411f-96b1-4582f0e3f8a3.png">